packaging/ebiso.spec

```diff
@@ -0,0 +1,48 @@
+%define bindir /usr/bin
+%define rpmrelease  %nil
+
+
+### Work-around the fact that OpenSUSE/SLES _always_ defined both :-/
+%if 0%{?sles_version} == 0
+%undefine sles_version
+%endif
+
+Name:			ebiso
+Version:		0.1.1
```

Comment:
@gozora that would not work I am afraid as version would not be defined at this point. What I use to do is edit the spec file via the Makefile (via a rewrite rule - see the rear Makfile for an example).